src/cc/bcc_syms.cc

```diff
@@ -315,9 +317,7 @@ bool ProcSyms::Module::find_name(const char *symname, uint64_t *addr) {
   return false;
```

Comment:
This part (`find_name`) is probably broken as well. But I have no idea how to fix. Given the binary offset, is it even possible to map it back to the correct memory load section?